auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/external/DeltaLoginController.kt

```diff
@@ -151,6 +145,13 @@ class DeltaLoginController(
             return call.respondRedirect(oauthClientLoginRoute(ssoClientMatchingEmailDomain.internalId, formUsername))
         }
 
+        if (password.isNullOrEmpty()) return call.respondLoginPage(
+            client,
+            errorMessage = "Password is required",
+            errorLink = "#password",
+            username = formUsername,
+        )
```

Comment:
Should tests be updated to cover this case? E.g. update testLoginPostSSODomainRedirects to not include a password?

Reply:
Good idea